css/portal-dashboard/question-area.less

```diff
@@ -2,6 +2,8 @@
 
 .questionArea {
   flex-basis: auto;
+  overflow-y: auto;
```

Comment:
I'm wondering if this would be better in the .questionTextArea instead so we keep the question type, activity link, and teacher edition button in view, and we only scrolll the actual question text?

Reply:
Yeah, that does feel a little better with the header area that has the question type and links remaining at the top.

Reply:
A pushed a change that contains your suggestion.  I think it's better now.